src/main/java/firebase/sdk/dashboard/dao/UserDaoDatastore.java

```diff
@@ -0,0 +1,73 @@
+package firebase.sdk.dashboard.dao;
+
+import firebase.sdk.dashboard.data.SDK;
+import firebase.sdk.dashboard.data.SDKReleaseMetadata;
+import firebase.sdk.dashboard.data.VersionMetadata;
+import firebase.sdk.dashboard.data.Release;
+import firebase.sdk.dashboard.data.Platform;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.time.Instant;
+import java.io.IOException;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.CompositeFilter;
+import com.google.appengine.api.datastore.Query.CompositeFilterOperator;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.QueryResultIterable;
+
+/* Class to implement UserDao */
+
```

Comment:
Remove this extra line. Use the google-java-formatter.

Reply:
The whole file needs formatting.

Reply:
https://github.com/google/google-java-format. You can install and run this to format easily.